### ISSUE TITLE
"Fatal Error" occurred when adding the Social extension

### ISSUE THREAD

**Issue by zaneyu** [ThemeFuse/Unyson#301]
I was trying to add the Social extension in Unyson, and suddenly the WP website went down by showing the following line:

"Fatal error: Cannot redeclare class OAuthSignatureMethod_HMAC_SHA1 in /home/content/p3pnexwpnas05_data03/81/2365281/html/wp-content/plugins/unyson/framework/extensions/social/extensions/social-twitter/includes/OAuth.php on line 125"

My site is hosted by Godaddy and now I'm not able to manage it (say, entering the dashboard page) with the above error. I don't know if it is because the extension conflicts with some other plugins.

It seems the only way to "solve" the problem is removing the old one and build a new one. However I don't have a backup so if anyone knows other way please tell me.

**Comment by moldcraft** [comment]
@zaneyu Delete `/home/content/p3pnexwpnas05_data03/81/2365281/html/wp-content/plugins/unyson/framework/extensions/social` directory.

----

@valeriuzdrobau I think in `social/extensions/social-twitter/includes/OAuth.php` you should check if class already exists (maybe included by other plugin)

```php
if (!class_exists('OAuthSignatureMethod_HMAC_SHA1')):
class OAuthSignatureMethod_HMAC_SHA1 {
    ...
}
endif;
```

**Comment by zaneyu** [comment]
@moldcraft Thanks a lot! :)

**Comment by valeriuzdrobau** [comment]
I corrected [here](https://github.com/ThemeFuse/Unyson-Social-Extension/commit/cd51c80e4fb2728eae99f97d82df3c27828d1997), the problem will disappear in the next version.

**Comment by valeriuzdrobau** [comment]
Fixed in [v1.0.4](https://github.com/ThemeFuse/Unyson-Social-Extension/releases/tag/v1.0.4)
